### PR TITLE
Fix jicofo_lock is null when restored from storage

### PR DESCRIPTION
**Pull request by hshim432**
https://prosody.im/doc/modules/mod_muc
> Limit on number of rooms in-memory
> Prosody keeps recently active rooms cached in memory for faster access. The size of this cache defaults to 100 rooms and can be configured like this:
> 
> muc_room_cache_size = 100 -- the default
> When a room is pushed out of this cache by other more recently active rooms, its complete state is serialized to storage and removed from memory.
> 
> Public servers may want to monitor and possibly increase this setting if rooms are moved in and out of memory very often.

When room is serialized to storage, room.jicofo_lock is not saved.
https://github.com/bjc/prosody/blob/master/plugins/muc/muc.lib.lua#L1671-L1675

When the room is activated again, jicofo_lock is null and remains locked.

I fixed to save jicofo_lock.

### CODE REVIEW

File: twa/app/src/main/res/xml/shortcuts.xml [hunk 1]
```diff
@@ -1,16 +1 @@
-<!--
```
Comment (saghul) [inline twa/app/src/main/res/xml/shortcuts.xml:1]: Please undo this change.

**Review by saghul — CHANGES_REQUESTED** [review]
The change LGTM but there is an unrelated hunk, can you please undo that?

File: twa/app/src/main/res/xml/shortcuts.xml (hunk 1 above)
Comment (hshim432) [reply twa/app/src/main/res/xml/shortcuts.xml]: @saghul 
Please check again. Thank you.

**Review by saghul — APPROVED** [review]

**Comment by damencho** [comment]
Jenkins test this please.

**Comment by saghul** [comment]
I don't see how the test failure is related. @damencho does this LGTY?
